CONTRIBUTING.md

```diff
@@ -2,6 +2,10 @@
 
 Prometheus uses GitHub to manage reviews of pull requests.
 
+* Every commit *MUST* signed off on the [DCO](https://github.com/dcoapp/app#how-it-works)
```

Comment:
Expand the acronym to make it a little more SEO, screen reader, and mobile screen friendly.

```suggestion
* Every commit *MUST* signed off on the [Developer Certificate of Origin](https://github.com/dcoapp/app#how-it-works) (DCO).
```

Reply:
Should it really be "on the" not "with the"?

Reply:
Agreed

Reply:
> Should it really be "on the" not "with the"?

So I believe "on the" does actually could make sense

But maybe
"Every commit *MUST* be signed off to agree to the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). [How does it work?](https://github.com/dcoapp/app#how-it-works)"

Reply:
Yeah, that sounds even better!